tests/test_sign_cmd.py

```diff
@@ -43,6 +44,45 @@ def test_sign_tx_short_tx(backend, scenario_navigator):
     assert check_signature_validity(public_key, der_sig, transaction)
 
 
+# In this test we send to the device a transaction to trig a blind-signing flow
+# The transaction is short and will be sent in one chunk
+# We will ensure that the displayed information is correct by using screenshots comparison
+def test_sign_tx_short_tx_blind_sign(firmware, navigator, backend, scenario_navigator, test_name, default_screenshot_path):
+    if not firmware.is_nano:
```

Comment:
Could be better/clearer to use the reverse:
```
    if firmware.is_nano:
        pytest.skip("Not supported on Nano devices")
```

Reply:
👍